swift/Sources/SwiftNoosphere/Noosphere.swift

```diff
@@ -166,3 +166,22 @@ public func nsSphereAuthorityAuthorizationRevoke(_ noosphere: OpaquePointer!, _
         handler.contents(error)
     }
 }
+
+public typealias NsSphereAuthorityListAuthorizationsHandler = (OpaquePointer?, slice_boxed_char_ptr_t?) -> ()
```

Comment:
Conventionally, verb should come _after_ `Authorizations` e.g., `NsSphereAuthorityAuthorizationsListHandler`